.github/workflows/gradle.yml

```diff
@@ -0,0 +1,16 @@
+name: Java CI
+
+on: [push]
```

Comment:
Would it also be beneficial to run this on PRs?

Reply:
Uh yes it probably should be changed to PR. Check action syntax